modules/tinymce/CHANGELOG.md

```diff
@@ -4,14 +4,16 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## Unreleased
+## 6.1.1 - 2022-07-27
 
 ### Fixed
-- Invalid special elements were not cleaned up correctly during sanitization #TINY-8780
-- An exception was thrown when deleting all content if the start or end of the document had a `contenteditable="false"` element #TINY-8877
-- When a sidebar was opened using the `sidebar_show` option, its associated toggle button was not highlighted #TINY-8873
-- The `autolink` plugin when converting a URL to a link did not fire an `ExecCommand` event, nor did it create an undo level #TINY-8896
-- Worked around a Firefox bug whereby cookies weren't available inside the editor content #TINY-88916
+- Invalid special elements were not cleaned up correctly during sanitization. #TINY-8780
+- An exception was thrown when deleting all content if the start or end of the document had a `contenteditable="false"` element. #TINY-8877
+- When a sidebar was opened using the `sidebar_show` option, its associated toolbar button was not highlighted. #TINY-8873
+- When converting a URL to a link, the `autolink` plugin did not fire an `ExecCommand` event, nor did it create an undo level. #TINY-8896
+- Worked around a Firefox bug which results in cookies not being available inside the editor content. #TINY-8916
+- When pasted, `<pre>` formatted text remained `<pre>` formatted and did not take on the properties of material it was pasted in to. #TINY-8860
```

Comment:
This doesn't look accurate based on the original bug... it's that the pre formatting was lost entirely, as in it failed to either merge or replace correctly.

Also, as a side note, this is going to conflict as that PR has not been merged yet.

Reply:
> This doesn’t look accurate based on the original bug and the associated fix...

FWIW, I wrote that up based on the behaviour of [this fiddle](https://fiddle.tiny.cloud/eiiaab).

When I copy and paste as instructed, I end up with 

Some **str**

`copy me`

**ong** content

> it's that the pre formatting was lost entirely, as in it failed to either merge or replace correctly.

So, something like as follows?

When pasted, `<pre>` formatted text lost its `<pre>` formatting (as expected) but did not take on the properties of material it was pasted in to. #TINY-8860

Reply:
No, that's still saying the wrong thing that was fixed. The original is the best way I can describe it:
```
- Pasting `<pre>` content into a `<pre>` block that had inline styles or was noneditable now correctly merges content #TINY-8860
```
So to try to clarify further, it's that the pasted content was either unwrapped when it shouldn't have been (the noneditable case) in which case it would replace the old element but since it was unwrapped it lost the `<pre>` element entirely (this is the one we mainly care about as it's what's in the 6.1 known issues). Or the other case was that if you pasted a `<pre>` element into a `<strong>` within a `<pre>` then it would **not** unwrap and instead of merging it would split the text.

Reply:
So basically it's just about the fact that pre elements were not merged correctly when pasting and that's what we want this to be focused on.

Reply:
I’m adding a new suggestion for this line. It’s a minor copy-edit of the original entry. Also, thanks for the explicit point about this being a fix to the 6.1 known issue: I’ve added a note to this effect in my `6.1.1-release-notes.adoc` skeleton.